cms/djangoapps/contentstore/views/item.py

```diff
@@ -552,7 +552,10 @@ def _duplicate_item(parent_usage_key, duplicate_source_usage_key, user, display_
         category = dest_usage_key.block_type
 
         # Update the display name to indicate this is a duplicate (unless display name provided).
-        duplicate_metadata = own_metadata(source_item)
+        duplicate_metadata = {}
```

Comment:
Yeah, the conversion to-from json is an ongoing confusion and problem which I don't think we've systematically encapsulated at the right places. There are probably other places which have similar issues. imho, I wish field data used python native types and not json and the only conversion to/from json would happen only at serialization to external services/clients.